lib/ui/platform_dispatcher.dart

```diff
@@ -274,13 +274,15 @@ class PlatformDispatcher {
     assert(!_views.containsKey(id));
     _views[id as int] = _createView(id);
     _viewConfigurations[id] = const _ViewConfiguration();
+    _invoke(onMetricsChanged, _onMetricsChangedZone);
```

Comment:
I'm fine with doing this as a temporary hack, but really the embedder API's `FlutterAddViewInfo` should have a view metrics member and `FlutterEngineAddView` should send window metrics. /cc @dkwingsmt

Reply:
Agreed.

Reply:
Left a TODO so we don't forget. Are you ok with submitting this then? Would make experimenting with the prototype a little easier...